packages/okta-react/test/e2e/harness/package.json

```diff
@@ -27,7 +27,7 @@
     "build:test": "rimraf e2e/dist && babel e2e/ -d e2e/dist",
     "kill:port": "kill -9 $(lsof -t -i:8080 -sTCP:LISTEN) || true",
     "pretest": "yarn kill:port && ./node_modules/.bin/webdriver-manager update --gecko false && yarn build:test",
-    "wait:server": "yarn start & ./node_modules/.bin/wait-on http://localhost:8080",
+    "wait:server": "BROWSER=none yarn start & ./node_modules/.bin/wait-on http://localhost:8080",
```

Comment:
When running end-to-end tests in React, we're now telling the dev server to **not** open the browser when it loads.